docs/microservices/logging-monitoring.md

```diff
@@ -23,10 +23,14 @@ ms.custom:
 
 This article describes best practices for monitoring a microservices application that runs on Azure Kubernetes Service (AKS).
 
-In any complex application, at some point something will go wrong. In a microservices application, you need to track what's happening across dozens or even hundreds of services. To make sense of what's happening, you must collect telemetry from the application. Telemetry can be divided into *logs* and *metrics*.
+In any complex application, at some point something will go wrong. In a microservices application, you need to track what's happening across dozens or even hundreds of services. To make sense of what's happening, you must collect telemetry from the application. Telemetry can be divided into _logs_, _traces_ and _metrics_.
 
 **Logs** are text-based records of events that occur while the application is running. They include things like application logs (trace statements) or web server logs. Logs are primarily useful for forensics and root cause analysis.
 
+**Traces**, also called _Operations_ connect the steps of a single request across multiple calls within and across microservices. They can provide structured observability into the interactions of system components. Traces can begin early in the request process like within the UI of an application and propagate through network services and across a network of microservices that handle the request.
+
+- **Spans** are units of work within a trace. Each span is connected with a single trace and can be hested with other spans. They often correspond to individual _requests_ in a cross-service operation but can also define work in individual components within a service. Spans also track outbound calls from one service to another (sometimes these are called _dependency_ records).
```

Comment:
```suggestion
- **Spans** are units of work within a trace. Each span is connected with a single trace and can be nested with other spans. They often correspond to individual _requests_ in a cross-service operation but can also define work in individual components within a service. Spans also track outbound calls from one service to another (sometimes these are called _dependency_ records).
```

Reply:
In addition, I think the sentence "Each span is connected with a single trace and can be hested with other spans." is a bit confusing and jargony.